collectors/logsmanagement.plugin/plugin_logsmanagement_systemd.c

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "plugin_logsmanagement_systemd.h"
+
+void systemd_chart_init(struct File_info *p_file_info, struct Chart_meta *chart_meta){
+    chart_meta->chart_data_systemd = callocz(1, sizeof (struct Chart_data_systemd));
+    chart_data_systemd_t *chart_data = chart_meta->chart_data_systemd;
+
+    /* Number of lines - initialise */
+    chart_data->st_lines = rrdset_create_localhost(
+            (char *) p_file_info->chart_name
+            , "lines parsed"
+            , NULL
+            , "lines parsed"
+            , NULL
+            , "Log lines parsed"
+            , "lines/s"
+            , "logsmanagement.plugin"
+            , NULL
+            , NETDATA_CHART_PRIO_LINES
+            , p_file_info->update_every
+            , RRDSET_TYPE_AREA
+    );
+    // TODO: Change dim_lines_total to RRD_ALGORITHM_INCREMENTAL
+    chart_data->dim_lines_total = rrddim_add(chart_data->st_lines, "Total lines", NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+    chart_data->dim_lines_rate = rrddim_add(chart_data->st_lines, "New lines", NULL, 1, 1, RRD_ALGORITHM_INCREMENTAL);
+
+    /* Syslog priority value - initialise */
+    if(p_file_info->parser_config->chart_config & CHART_SYSLOG_PRIOR){
+        chart_data->st_prior = rrdset_create_localhost(
+                (char *) p_file_info->chart_name
+                , "priority value"
+                , NULL
+                , "priority"
+                , NULL
+                , "Priority Value"
+                , "priority values/s"
+                , "logsmanagement.plugin"
+                , NULL
+                , NETDATA_CHART_PRIO_SYSLOG_PRIOR
+                , p_file_info->update_every
+                , RRDSET_TYPE_AREA
+        );
+        for(int j = 0; j < 192; j++){
```

Comment:
Why did you select the magic number 192? Considering it is used immediately below, please, create a `define` with a name that can help us to understand this better.

Reply:
Done.